packages/google_atlas/lib/google_atlas.dart

```diff
@@ -1,3 +1,4 @@
 library google_atlas;
 
 export 'src/google_atlas.dart';
+export 'src/utils/utils.dart';
```

Comment:
this doesn't make sense to me. Why would we want to export the internal utility functions? @jorgecoca